src/scripts/main.js

```diff
@@ -1,3 +1,58 @@
 'use strict';
 
-// write code here
+const table = document.querySelector('.field');
+const addRow = document.querySelector('.append-row');
+const delRow = document.querySelector('.remove-row');
+const addCol = document.querySelector('.append-column');
+const delCol = document.querySelector('.remove-column');
+const max = 10;
+const min = 2;
```

Comment:
it's better to use UPPERCASE when you use variables like this (shows that variables are constants)
```suggestion
const MAX = 10;
const MIN = 2;
```

Reply:
Also add meaningful names, because its not clear MAX of what?